packages/client/src/helpers/client/interceptors/authentication/token-providers/TokenData.ts

```diff
@@ -1,3 +1,5 @@
+import type { Data } from './types/TokenData.types';
```

Comment:
I would call the `Data` type `ITokenData` as it is more specific. Also, class TokenData should implement the Data interface like this:

```js
class TokenData implements ITokenData {
...
```

Reply:
done :)